source/geod24/concurrency.d

```diff
@@ -1067,6 +1067,9 @@ protected:
         /// The mutex with which this condition will be associated.
         private Mutex _mutex;
 
+        /// When notify() is called, this value is true.
+        private bool _notified;
```

Comment:
You don't need to use mutexes for simple scalar values like bool, you can just use CAS

Reply:
Oh, good idea. I'll give it a try.

Reply:
Since FiberCondition.notify() and FiberScheduler.dispatch() need to synchronize, we need to use the mutex from the FiberScheduler.

Reply:
I did a test. When changing to CAS, it was confirmed that it worked normally. I'll change it to the code below.
```
        private @property void notified (bool value) nothrow
        {
            if (this._notified == value) return;
            while (!cas(&this._notified, !value, value)) { Thread.yield(); }
        }
```

Reply:
Don't `Thread.yield()` in the `cas`, just loop.